design/features/confirmation-email-for-form-fillers/README.md

```diff
@@ -0,0 +1,57 @@
+# Confirm that a form's been submitted (form fillers)
+
+## Status
+
+Date created: *2023-11-06*  
+
+* Epic trello card: https://trello.com/c/0svlNImf/1089-epic-confirm-that-a-forms-been-submitted  
+* Mural working board: https://app.mural.co/t/gaap0347/m/gaap0347/1688563053078/f59d07f0cd2dca526768595d30228b4b0188ea32?sender=ue1ef9fc3c2ad3697c3c93132 
+___
+
+## Contents
+
+- [Confirm that a form's been submitted (form fillers)](#confirm-that-a-forms-been-submitted-form-fillers)
+  - [Status](#status)
+  - [Contents](#contents)
+  - [What](#what)
+  - [Why](#why)
+  - [Hypothesis](#hypothesis)
+
+___
+
+<br>
+
+## What
+
+This feature introduces new functionality to the form filler's journey, allowing them to request an email confirming that they've successfully submitted a form. 
+
+### User stories
+
+**As a form creator**, I need to know how the content I write will be used, so that I know what information to include to help our form fillers.
+
+**As a form filler**, I need confirmation my form has been submitted, so that I feel confident that I have completed my task.
+
+**As a form filler**, I need a record of my submission, so that I have a reminder of when I sent my form, and know how to chase it up if I haven’t heard back.
+
+<br>
+
+## Why
+
+Currently, once a form is submitted the form filler is shown a confirmation screen telling them that their form has been submitted. This screen also includes information about what will happen next, in terms of timelines and expectation management. 
+
+We know that some of our forms also use this content to set out next steps that a form filler needs to complete. However, these steps might not be time sensitive or they might require more information than a form filler has at that moment. Having a record of these steps in an email will help form fillers return to complete these tasks at a later time.  
+
+## Hypothesis
+
+We believe that by offering form fillers a way to request an email confirming that they've submitted the form - which will include the form name, date and time of submission, as well as information provided by the form creators ('what happens next' information and contact details for support) - we'll be able to:
+
+* reassure form fillers
+* reduce the number of contacts to call centres or processing staff, allowing them to complete the task for the form filler.  
+
+<br>
```

Comment:
why are there br tags here?

Reply:
again to add padding and prettify the rich version